graphsdk/src/main/java/com/microsoft/graph/generated/BaseDirectoryObjectReferenceRequest.java

```diff
@@ -60,26 +60,27 @@ public IDirectoryObjectReferenceRequest select(final String value) {
      */
     public IDirectoryObjectReferenceRequest expand(final String value) {
         getQueryOptions().add(new QueryOption("$expand", value));
-        return (DirectoryObjectReferenceRequest)this;
+        return (DirectoryObjectReferenceRequest) this;
```

Comment:
Appears to be an artifact of an existing formatting error in the template. I don't stress this stuff, but it's there.